compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -3258,7 +3258,7 @@ OMR::Power::CodeGenerator::addMetaDataFor64BitFixedLoadLabelAddressIntoReg(
       TR::Instruction **q)
    {
 
-   if (!self()->comp()->compileRelocatableCode())
+   if (!self()->comp()->compileRelocatableCode() && !self()->comp()->isOutOfProcessCompilation())
```

Comment:
Copying over a comment from @ymanton regarding an alternative to this change here:

https://github.com/eclipse/omr/pull/4441#pullrequestreview-319337507

Reply:
@ymanton Here's an alternative solution I had thought of for all four cases here:

The code changes in this PR have a common theme in that we are trying to prevent the codegen from emitting data inside instructions that will be relocated externally. So I had initially thought that a common solution for all four of the scenarios here would be to introduce a new query such as query`emitDataForExternallyRelocatableInstructions`. In the context of the routines themselves, this would make sense and we can leave in a comment when defining the query about why on Power we must not emit data.

Reply:
Makes sense.

Reply:
New query introduced and appropriately used here: 200634121c31bd0a24e7130ee36375fa8e3fb34e